src/components/molecules/TextView.tsx

```diff
@@ -47,7 +49,15 @@ const useStyles = makeStyles((theme: Theme) => ({
 }));
 const TextView: FC<IProps> = ({ data, segmentText, roadNumber }) => {
   const classes = useStyles();
+  const { t, i18n } = useTranslation();
   const { items } = data;
+  const accidentsOccurredEls = [
```

Comment:
Please explain here what you did (I understand it is due to English vs Hebrew differences, but it is not clear from brief reading).
Also, this better be refactored to a separate component (can be included in the same file, before `TextView`)